server/integration-test/src/test/java/com/oceanbase/odc/service/pldebug/PLDebugTest.java

```diff
@@ -107,7 +107,8 @@ public void test_call_procedure() throws Exception {
         params.add(paramC);
         DBProcedure odcProcedure = DBProcedure.of(null, "PROC", params);
 
-        CallProcedureCallBack procedureCallBack = new CallProcedureCallBack(odcProcedure, 30, new OracleSqlBuilder());
+        CallProcedureCallBack procedureCallBack =
+                new CallProcedureCallBack(odcProcedure, 30, new OracleSqlBuilder(), null);
```

Comment:
There is too much code with constructor parameters set to `null`. Try creating a new constructor that does not need pass `PLDebugODPSpecifiedRoute` and set it to false by default.

Reply:
![image](https://github.com/user-attachments/assets/d67aa97f-8d7b-4ca0-bd6c-5c82c91faee0)
like that？